src/service/IssueReporter.ts

```diff
@@ -364,6 +364,14 @@ class IssueReporter {
     const isSecure = +port === 443;
     const url = `http${isSecure ? 's' : ''}://${host}${isSecure ? '' : `:${port}`}/${tag}`;
     const body = Array.isArray(payload) ? `[${payload.map(this._boundParseLoggerBody).join(',')}]` : this._parseLoggerBody(payload as Record<string, any>);
+    const waitfn = (attempt: number, initial = 5000, maxWait = 50000) => {
```

Comment:
Can we have the `waitFn` outside of the `_sendDebugToGrafana` method ?
And maybe we can call it `_computeNextAttemptDelay`.